From: Xavier Claessens <xavier.claessens@collabora.com>
Date: Tue, 25 Sep 2018 09:33:15 -0400
Subject: Meson: Do not run tests/refcount with --tap

This is a partial revert of a change introduced in cbc7fbbf7.

Origin: upstream, 2.59.0, commit:7c8e1f15cc500d2a24742435ae73a54856d7bfe6
---
 tests/refcount/meson.build | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/tests/refcount/meson.build b/tests/refcount/meson.build
index fe341ca..3a2072d 100644
--- a/tests/refcount/meson.build
+++ b/tests/refcount/meson.build
@@ -55,6 +55,5 @@ foreach test_name, extra_args : refcount_tests
   suite = ['refcount'] + extra_args.get('suite', [])
   timeout = suite.contains('slow') ? 120 : 30
   # FIXME? TESTS_ENVIRONMENT = LIBCHARSET_ALIAS_DIR=$(top_builddir)/glib/libcharset
-  test(test_name, exe, env : test_env, timeout : timeout, suite : suite,
-    args : ['--tap'])
+  test(test_name, exe, env : test_env, timeout : timeout, suite : suite)
 endforeach
